blackduck/Client.py

```diff
@@ -56,14 +57,15 @@ def __init__(self, base_url, timeout, retries, verify):
 
     def request(self, method, url, **kwargs):
         kwargs['timeout'] = self._timeout
-        if method.lower() == 'get':
-            # set default media type if not provided
-            headers = {
-                'accept': "application/json",
-                'content-type': "application/json"
-            }
-            headers.update(kwargs.pop('headers', dict()))
-            kwargs['headers'] = headers
+
```

Comment:
I can set the default media type for 'get' and 'post'

Reply:
After testing, 'content-type': "application/json" is needed to successfully POST.